packages/domain-search/src/hooks/use-policy-badges.tsx

```diff
@@ -0,0 +1,23 @@
+import { type ReactNode, useMemo } from 'react';
+import { DomainSuggestionBadge } from '../ui';
+import { usePolicyNotices } from './use-policy-notices';
+
+export const usePolicyBadges = ( domainName: string ) => {
```

Comment:
I don't think we need this extra hook. Can we modify `use-suggestion-badges` to include this logic as well?

Reply:
That was my thought at first, but then I've noticed [there's a case where we want them separated](https://github.com/Automattic/wp-calypso/pull/105532/files#diff-430392a09b363824074ae1594011fe7630347a03585d96f8c0af8ed2eabf3533R34-R44). In the featured results, we're not showing suggestion badges when the domain is available. Let me know if you think I should refactor that part.

Reply:
I've removed the `usePolicyNotices` in https://github.com/Automattic/wp-calypso/pull/105532/commits/7160721c9db3a749e2d7d51ba763c220574c11a9. 👍